### PR TITLE
Utility breakpoints

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

File: utilities/_u-alignments.scss [hunk 1]
```diff
@@ -48,6 +76,12 @@
   justify-content: center;
 }
 
+@include generate-at-breakpoints('.u-align-v-block-middle-flexbox', $alignments-breakpoints) {
```
Comment (AlecRaeside) [inline utilities/_u-alignments.scss]: There are a couple of lines over 80 chars

**Comment by AlecRaeside** [comment]
LGTM

File: utilities/_u-columns.scss [hunk 2]
```diff
@@ -74,22 +103,49 @@ $column-divider-colour:       $colour-text-base !default;
...
 %u-columns--divider,
-.u-columns--divider {@include to-rem(column-rule, $column-divider-thickness $column-divider-style $column-divider-colour);}
\ No newline at end of file
+.u-columns--divider {@include to-rem(column-rule, $column-divider-thickness
+  $column-divider-style $column-divider-colour);}
+
+@include generate-at-breakpoints('.u-columns--divider',
+  $column-breakpoints) {
+  @include to-rem(column-rule, $column-divider-thickness $column-divider-style
+    $column-divider-colour);
```
Comment (kllevin) [inline utilities/_u-columns.scss:144]: should these be aligned?

File: utilities/_u-display.scss [hunk 3]
```diff
@@ -71,6 +109,12 @@ $display-js-is-off-class:		no-js !default;
 	visibility: hidden;
 }
 
+@include generate-at-breakpoints('.u-hide',
+  $display-breakpoints) {
+  display: none;
+	visibility: hidden;
+}
```
Comment (kllevin) [inline utilities/_u-display.scss:111]: fix alignment? or gh is displaying weird.

**Pull request merged**